code/game/mecha/working/working.dm

```diff
@@ -2,14 +2,16 @@
 	internal_damage_threshold = 40
 	exit_delay = 10
 	enter_delay = 20
+	light_power = 9
+	light_range = 10
 
 /obj/mecha/working/Move()
 	. = ..()
 	if(.)
 		collect_ore()
 
 /obj/mecha/working/proc/collect_ore()
-	if((locate(/obj/item/mecha_parts/mecha_equipment/hydraulic_clamp) in equipment) || (locate(/obj/item/mecha_parts/mecha_equipment/orebox_manager) in equipment))
```

Comment:
Why?

Reply:
Because it's being removed from the Clarke as it's now a Ripley sub-type and can fit a hydraulic clamp.

Reply:
No, the ore holder is meant to be spawned on Clarke, it's the whole idea of it - mining mech.

Reply:
According to whom? /VG/station devs spirited the Clarke originally and it doesn't have a pre-installed ore box. The idea is that it's a utility mech that can do either task rather than becoming a paperweight the second mining is finished.